src/components/users/LoginForm/login-form.scss

```diff
@@ -0,0 +1,34 @@
+@import "../../../style/common/constants";
+@import "../../../style/common/typography";
+
+.login-form {
+  display: flex;
+  flex-direction: column;
+  height: 100%;
+  align-items: center;
+  align-content: center;
+}
+
+.login-form-input-container {
```

Comment:
Remember that when using scss you can have nesting inside the classes. For example you could have the login-form class as a wrapper for the form, and then inside this class use selectors such as label, button, input, etc to reach to the elements you want to style inside the form. The code would look like
`.login-form {
/*your form styles*/
input {
/*your input styles inside the form*/
},
button {
/*your button styles inside the form*/
}
}`